resources/README.md

```diff
@@ -0,0 +1,76 @@
+# Eventing-kafka Helm Chart
+
+This helm chart installs the eventing-kafka knative eventing implementation.  When installing make sure to provide the appropriate values file for the
```

Comment:
There's nothing else in knative that requires helm, so it would be nice to not have that requirement? typically we have the config/* with a standard layout (well, at least we try to).

Reply:
Are there any examples which don't use ko for the image builds elsewhere in knative? Creating release yaml independent of helm will be straightforward, but I'm not quite sure what the right templating approach would be for yaml in the config folder. Ko doesn't support building with the c dependencies.

Reply:
My understanding was that there was a way to do c deps with ko:
https://github.com/tektoncd/pipeline/blob/c376874b1d4ebb5c5f38f6e305382b54a3766a64/tekton/publish.yaml#L79-L86

@mattmoor

Reply:
Maybe I'm missing something, but the ko build still runs in a local context where the c dependencies must already be installed.  The output image must also have the c dependencies installed and be of the same os and architecture that the go binary was built against.  Is the intention to do some static linking instead somehow? I'm not a cgo expert, but I can get other people from my team to chime in and take a look if you guys think it should be workable

Reply:
Could we split this into a separate issue to continue as well?  I'm not quite sure how it will work together with a release build which we'll work on next

Reply:
A teammate just gave me some good news! https://github.com/confluentinc/confluent-kafka-go#librdkafka librdkafka is now bundled in with the go client, so we might be ko friendly as soon as we upgrade to this version of the kafka go client.

Reply:
I've spent some time working on this, and looks like we should be ok once the confluent library version is upgraded.  Then we can use the bundled c libraries that are included and don't have any additional upfront installation steps required.  Cross compilation is still a challenge, but the knative release build itself could end up being ok with ko I think. Would be great to close this pr and start working on this in a new issue. It's quite cumbersome to keep things in sync and do new active development on this huge pr

Reply:
I am also for removing dependency on helm. Could we build it without make?

Reply:
We can start a discussion on solution options here https://github.com/knative-sandbox/eventing-kafka/issues/3